bigframes/ml/core.py

```diff
@@ -136,6 +136,14 @@ def evaluate(self, input_data: Optional[bpd.DataFrame] = None):
 
         return self._session.read_gbq(sql)
 
+    def arima_evaluate(self, show_all_candidate_models: bool = False):
+        # TODO: validate input data schema
```

Comment:
What's this TODO about? Please explain more in the comment before merging this PR.